### ISSUE TITLE
Global class for button Cancel & Save

### ISSUE THREAD

**Issue by sonit1810** [divppc/Physio-App#67]
Ihor, another request from Justin in Team Member page

"2. add cancel button (all edit, save, cancel buttons should be same for all pages across whole site)"

So can you make global class for button Cancel and Save as same style in Profile page ? So i can use it for other buttons on Setting pages

**Comment by divppc** [comment]
should i do that if you need another, new template for team member page? #69 
in the new design we don't have this buttons

**Comment by sonit1810** [comment]
I meant "Edit" and "Save" button, they are different in Profile page now, we need make them the same for all page

![image](https://user-images.githubusercontent.com/8002065/30528082-b4f64c7e-9c59-11e7-8279-554bf9bea5d7.png)

![image](https://user-images.githubusercontent.com/8002065/30528103-d92dfff6-9c59-11e7-929e-d78e62a0bb7e.png)


![image](https://user-images.githubusercontent.com/8002065/30528116-f25595e8-9c59-11e7-8ede-77109e1fa22d.png)

**Comment by divppc** [comment]
Hi Sonit. Should i use this styles of Cancel and Save buttons on another pages?
https://user-images.githubusercontent.com/8002065/30528116-f25595e8-9c59-11e7-8ede-77109e1fa22d.png

**Comment by sonit1810** [comment]
No, please use style from general setting page, we have edit icon with
pencil, and save button at bottom, that's style are ok. Because else i will
need update many pages. Only profile page has different style.

Thanks!

**Comment by divppc** [comment]
ok, so on the profile page i'll make same styles as in general settings page, green text with pencil icon

**Comment by divppc** [comment]
Son, what about cancel page on Personal profile? if i change Save button they will be different

**Comment by divppc** [comment]
please check here http://physio.div.agency/personal-profile.html

.edit-btn and .save-btn are global classes
